packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -526,7 +526,13 @@ export const createAzureCommunicationChatAdapter = async ({
   credential,
   threadId
 }: AzureCommunicationChatAdapterArgs): Promise<ChatAdapter> => {
-  return _createAzureCommunicationChatAdapterInner(endpointUrl, userId, displayName, credential, threadId);
+  return _createAzureCommunicationChatAdapterInner(
+    endpointUrl,
+    userId,
+    displayName,
+    credential,
+    new Promise((r) => r(threadId))
```

Comment:
Should this be async? the regular chat adapter I don't think should be affected

Reply:
Agree, i've added separate function for call and chat scenario